apps/firebase-api/src/app/users.ts

```diff
@@ -0,0 +1,56 @@
+import admin from 'firebase-admin';
+import { logger } from 'firebase-functions';
+
+const getUserRef = (userId) =>
+  admin.firestore().collection('users').doc(userId);
+
+const addUNumToAllUsers = () => {
+  let uNum = 1;
+
+  const iterateAllUsers = async (nextPageToken?) => {
+    // List batch of users, 1000 at a time.
+    try {
+      const listUsersResult = await admin.auth().listUsers(1, nextPageToken);
```

Comment:
This would only give you 1 user. https://firebase.google.com/docs/reference/admin/node/admin.auth.Auth-1#listusers